### DISCUSSION TITLE
support screen-sharing a single window

### DISCUSSION THREAD

**Discussion by Kha** [YaLTeR/niri#336] [Ideas]
I didn't see a tracking issue/discussion for this yet. This is especially helpful for widescreen users where sharing the full screen is bad for attendees with more traditional resolutions and usually requires adjusting the font size.

**Comment by YaLTeR** [comment]
The issue is here: https://github.com/YaLTeR/niri/issues/116

**Reply by Kha** [reply]
Thanks!
